Zend/Optimizer/zend_func_infos.h

```diff
@@ -60,7 +62,7 @@ static const func_info_t func_infos[] = {
     F1("timezone_open", MAY_BE_OBJECT|MAY_BE_FALSE),
     F1("timezone_name_get", MAY_BE_STRING),
     F1("timezone_name_from_abbr", MAY_BE_STRING|MAY_BE_FALSE),
-    F1("timezone_transitions_get", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_KEY_STRING|MAY_BE_ARRAY_OF_ANY|MAY_BE_FALSE),
+    F1("timezone_transitions_get", MAY_BE_ARRAY|MAY_BE_ARRAY_KEY_LONG|MAY_BE_ARRAY_KEY_STRING|MAY_BE_ARRAY_OF_ANY|MAY_BE_ARRAY_OF_REF|MAY_BE_FALSE),
```

Comment:
We could make some of the changes lines more specific, but at least this is conservatively correct...